### DISCUSSION TITLE
When could the openglobus support 3dtiles and gltf?

### DISCUSSION THREAD

**Discussion by ywszxdd** [openglobus/openglobus#673] [General]
When could the openglobus support 3dtiles and gltf?

**Comment by Zemledelec** [comment]
Hi there! Gltf and [3dTiles ](https://github.com/openglobus/openglobus/issues/625) are two big tasks. First, we have a roadmap(issues) 3dTiles, and point clouds support on the roadmap. I can't say exactly "when". It depends on my or the contributors' free time. Definitely, such wonderful technology from Cesium - 3DTiles will be there sooner or later:) Cheers!  The same story with gltf, but .obj (with BRDF) first, I think...

**Comment by Zuzon** [comment]
gltf already there

**Comment by Zemledelec** [comment]
gltf examples:
https://sandbox.openglobus.org/examples/curiosity
https://sandbox.openglobus.org/examples/windFarm
